packages/xrpl/src/utils/createXchainPayment.ts

```diff
@@ -0,0 +1,40 @@
+import { XrplError } from '../errors'
+import { Payment } from '../models'
+import { Memo } from '../models/common'
+
+import { convertStringToHex } from './stringConversion'
+
+/**
+ * Computes the complete list of every balance that changed in the ledger
+ * as a result of the given transaction.
+ *
+ * @param payment - The initial payment transaction. If the transaction is
+ * signed, then it will need to be re-signed. There must be no more than 2
+ * memos, since one memo is used for the sidechain destination account.
+ * @param destAccount - the destination account on the sidechain.
+ * @returns A cross-chain payment transaction, where the mainchain door account
+ * is the `Destination` and the destination account on the sidechain is encoded
+ * in the memos.
+ * @throws XrplError - if there are more than 2 memos.
+ * @category Utilities
+ */
+export default function createXchainPayment(
```

Comment:
Naming wise I think it's better to explicitly write "Cross" rather than "X" since X is actually meaningful sometimes in our libraries (Ex. xAddress)

Also, "chain" should be capitalized.

I'd recommend to rename this function to `createCrossChainPayment`

Reply:
I agree. Let's use "Cross" instead to avoid ambiguity.